### PR TITLE
feat(Translation): Convert step title input to translatable

### PR DESCRIPTION
**Pull request by geoffreykwan**
## Changes
- Convert step title input to translatable input

## Test
- Make sure the step title is translatable

### CODE REVIEW

**Review by hirokiterashima — CHANGES_REQUESTED** [review]
1. Can we show the english text under the title like we do with other inputs?
Screenshot 2024-07-02 at 10 35 41 AM

2. The ```Step Title {{ nodePosition }}:``` and ``` Lesson Title {{ nodePosition }}:``` i18n strings appear to be removed from messages.xlf. Shouldn't they be translatable?

**Comment by geoffreykwan** [comment]
The English text now shows when authoring other languages. I moved the translations from the .html into the .ts file. I also fixed an issue with the width for all translatable inputs.

File: src/assets/wise5/authoringTool/node/edit-node-title/edit-node-title.component.ts [hunk 1]
```diff
@@ -4,21 +4,23 @@ import { TeacherProjectService } from '../../../services/teacherProjectService';
...
 
   ngOnChanges(): void {
-    this.isGroupNode = this.node.isGroup();
     this.nodeJson = this.projectService.getNodeById(this.node.id);
-    this.nodePosition = this.projectService.getNodePositionById(this.node.id);
+    this.isGroupNode = this.node.isGroup();
+    this.label =
+      (this.isGroupNode ? $localize`Lesson Title` : $localize`Step Title`) +
+      ' ' +
+      this.projectService.getNodePositionById(this.node.id);
```
Comment (hirokiterashima) [inline src/assets/wise5/authoringTool/node/edit-node-title/edit-node-title.component.ts]: Remove isGroupNode class variable and use this.node.isGroup() directly in the ternary expression?
```suggestion
    this.label =
      (this.node.isGroup() ? $localize`Lesson Title` : $localize`Step Title`) +
      ' ' +
      this.projectService.getNodePositionById(this.node.id);
```

**Review by hirokiterashima — APPROVED** [review]
LGTM. I think you can remove the isGroupNode class variable.

**Pull request merged**